src/components/FollowUserCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { Card, Image } from "react-bootstrap";
+import styled from "styled-components";
+
+import { MockUser } from "../types/mock-api-types";
+import FollowButton from "./FollowUserDialogueButton";
+
+const HeaderSection = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const FollowerUserCard = (props: { user: MockUser }) => {
+  const user = props.user;
+  return (
+    <Card>
+      <Card.Body>
+        <HeaderSection>
```

Comment:
u can use bootstrap styles here
```suggestion
        <section className="d-flex justify-content-between">
```